src/tests/modules/exec/sync.unlang

```diff
@@ -1,37 +1,33 @@
 #
 #  Sync calls return the response
 #
-update request {
-	&Tmp-String-0 := "%(exec_sync:/bin/sh -c 'echo hello')"
-}
+&Tmp-String-0 := "%(exec_sync:/bin/sh -c 'echo hello')"
 if (&Tmp-String-0 != 'hello') {
 	test_fail
 }
 
 #
 #  Call something which will take longer than the timeout
 #
-update request {
-	&Tmp-String-0 := "%(exec_sync:/bin/sleep 10)"
-}
+&request -= &Tmp-String-0
+&Tmp-String-0 := "%(exec_sync:/bin/sleep 10)"
 
 if (&Module-Failure-Message[*] != "Timeout running program - killing it and failing the request") {
 	test_fail
 }
 
-if (&Tmp-String-0 != "") {
+if (&Tmp-String-0) {
 	test_fail
 }
 
-update request {
-	&Tmp-String-0 := "%(exec_sync:/bin/sh $ENV{MODULE_TEST_DIR}/fail.sh)"
-}
+&request -= &Tmp-String-0
+&Tmp-String-0 := "%(exec_sync:/bin/sh $ENV{MODULE_TEST_DIR}/fail.sh)"
```

Comment:
You don't need to delete it, and then assign it using `:=`.  You can just use `:=`.  This is documented in the `edit.adoc` file.